### PR TITLE
[ Issue 13479 ] Fixed internal topic effect by InactiveTopicPolicy.

### LINKED ISSUE #13479: [bug](broker): brokerDeleteInactiveTopic should exclude pulsar internal topics
**Describe the bug**
On the Pulsar 2.8.1, we found that `brokerDeleteInactiveTopicsEnabled=true` will make pulsar internal topics like healthcheck deleted by inactivity.

截屏2021-12-24 上午10 35 10

**To Reproduce**
1. Set up pulsar cluster
2. Config `brokerDeleteInactiveTopicsEnabled=true` 
3. Monitor broker logs, you will get the healthcheck deleted by inactivity.

**Expected behavior**
InactiveTopics config should exclude some pulsar internal topics.

**Additional context**
Pulsar version: 2.8.1

### PR DESCRIPTION
**Pull request by mattisonchao**
Fixes #13479

Master Issue: #13479

### Motivation

see #13479

### Modifications

- make ``healthcheck``  internal topic as SystemTopic.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Does this pull request potentially affect one of the following parts:

none.

### Documentation

- [x] `no-need-doc`

### CODE REVIEW

**Comment by mattisonchao** [comment]
@codelipenghui  @ericsyh @eolivelli @BewareMyPower 

PTAL :) 

**Another things:**

Although this PR can fix the issue #13479,  But I have some advice as below:

1. ``healthcheck`` keyword name is common , maybe we can change this to some complex name.
2. I think we can collect all internal or system theme names into one container, because they are currently everywhere. Unclear and difficult to maintain.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java [hunk 1]
```diff
@@ -317,7 +318,7 @@ public void healthcheck(@Suspended AsyncResponse asyncResponse,
                             pulsar().getConfiguration());
 
 
-            topic = String.format("persistent://%s/healthcheck", heartbeatNamespace);
+            topic = String.format("persistent://%s/%s", heartbeatNamespace, HEALTH_CHECK_TOPIC_SUFFIX);
```
Comment (liudezhi2098) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java:321]: You may  Don’t need format

Comment (mattisonchao) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java:321]: Could you give me more information to help me know why i don't need format :)

**Comment by mattisonchao** [comment]
@codelipenghui   @eolivelli @BewareMyPower  @ericsyh  @Technoboy- 

PTAL :)

**Review by Technoboy- — APPROVED** [review]
LGTM

**Review by RobertIndie — APPROVED** [review]
LGTM.

**Review by codelipenghui — APPROVED** [review]

**Comment by codelipenghui** [comment]
@mattisonchao There are many conflicts with branch-2.8, could you please help push a new PR to fix the problem of branch-2.8?

**Comment by mattisonchao** [comment]
Sure, I will fix it ASAP

**Comment by mattisonchao** [comment]
@codelipenghui 

work done.
The new PR is #13816.

**Pull request merged**